tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -0,0 +1,129 @@
+    else:
+        pytest.fail("Matched number of HASH_SEED is not correct.")
+    return numeric_value
+
+
+def parse_ecmp_offset(outputs):
+    # Regular expression pattern to extract OFFSET_ECMP values (hexadecimal)
+    pattern = r'OFFSET_ECMP=(0x?[0-9a-fA-F]?)'
+
+    # Extracted values
+    extracted_values = []
+
+    for line in outputs.splitlines():
+        line = line.strip()
+        matches = re.findall(pattern, line)
+        if len(matches) == 1:
+            value = matches[0]
+            extracted_values.append(value)
+        elif len(matches) == 0:
+            continue
+        else:
+            pytest.fail("Matched number of OFFSET_ECMP is not correct.")
+    return extracted_values
+
+
+@pytest.mark.parametrize("parameter", ["", "restart_syncd", "reload"])
+def test_ecmp_hash_seed_value(duthosts, enum_rand_one_per_hwsku_frontend_hostname, parameter):
+    """
+    Check ecmp HASH_SEED
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    asic = duthost.facts["asic_type"]
+
+    if (asic != "broadcom"):
+        pytest.skip("Unsupported asic type: {}".format(asic))
+        return
+    if parameter == "":
+        for cmd in seed_cmd:
+            output = duthost.command(cmd, module_ignore_errors=True)["stdout_lines"][2].strip()
+            hash_seed = parse_hash_seed(output)
+            pytest_assert(hash_seed == '0xa', "HASH_SEED is not set to 0xa")
+    elif parameter == "restart_syncd":
+        duthost.command("docker restart syncd", module_ignore_errors=True)
+        logging.info("Wait until all critical services are fully started")
+        wait_critical_processes(duthost)
+        for cmd in seed_cmd:
+            output = duthost.command(cmd, module_ignore_errors=True)["stdout_lines"][2].strip()
+            hash_seed = parse_hash_seed(output)
+            pytest_assert(hash_seed == '0xa', "HASH_SEED is not set to 0xa")
+    elif parameter == "reload":
+        config_reload(duthost, safe_reload=True, check_intf_up_ports=True)
+        for cmd in seed_cmd:
+            output = duthost.command(cmd, module_ignore_errors=True)["stdout_lines"][2].strip()
+            hash_seed = parse_hash_seed(output)
+            pytest_assert(hash_seed == '0xa', "HASH_SEED is not set to 0xa")
+
+
+@pytest.mark.parametrize("parameter", ["", "restart_syncd", "reload"])
+def test_ecmp_offset_value(duthosts, enum_rand_one_per_hwsku_frontend_hostname, parameter):
+    """
+    Check ecmp HASH_OFFSET
+    """
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    asic = duthost.facts["asic_type"]
+
+    if (asic != "broadcom"):
+        pytest.skip("Unsupported asic type: {}".format(asic))
+        return
+    if parameter == "":
+        output = duthost.shell(offset_cmd, module_ignore_errors=True)['stdout']
+        offset_list = parse_ecmp_offset(output)
+        count_0xa = offset_list.count('0xa')
+        pytest_assert(count_0xa >= 67, "the count of 0xa OFFSET_ECMP is not correct.")
```

Comment:
67, this number may be different for different HWSKU/platform, we may consider to use a dict to hold the values then compare based on that.